README.md

```diff
@@ -91,6 +63,12 @@ This will start all the playground in watch mode.
 
 The project is built using [SolidJS](https://www.solidjs.com/). If you are new to SolidJS, we recommend checking out their [documentation](https://www.solidjs.com/docs) and [tutorials](https://www.solidjs.com/tutorial).
 
+### Suggested VSCode Extensions
+
+- [Tailwind CSS IntelliSense](https://tailwindcss.com/)
```

Comment:
Would it make sense for this link to go to the marketplace instead of the core tailwind website? Same goes for Prettier as well.